### ISSUE TITLE
Pause module doesn't allow continuation with v2

### ISSUE THREAD

**Issue by abadger** [ansible/ansible#11386]
Control-C is being trapped differently in v2 ansible which doesn't allow ansible's pause module to continue early.

```
$ cat pause.yml

---
- hosts: localhost
  tasks:
    - pause:
        prompt: "One"
    - pause:
        prompt: "Two"

$ ansible-playbook pause.yml
```

Hitting Control C at the pause prompt will cause ansible-playbook to exit.  In 1.9.1. this is displayed instead:

```
PLAY [localhost] ************************************************************** 

GATHERING FACTS *************************************************************** 
ok: [localhost]

TASK: [pause prompt='One'] **************************************************** 
[localhost]
One:
^C
Action? (a)bort/(c)ontinue: 
ok: [localhost]

TASK: [pause prompt='Two'] **************************************************** 
[localhost]
Two:
^C
Action? (a)bort/(c)ontinue: 
ok: [localhost]

PLAY RECAP ******************************************************************** 
localhost                  : ok=3    changed=0    unreachable=0    failed=0
```

**Comment by bcoca** [comment]
does it make sense to allow for "escape" to act as an interrupt? ctrl+c would still work but I think it is a 'nicer' way to do it.

**Comment by jimi-c** [comment]
Fixed by 9936c5f9f62310b312680b5a7ecf0642b70b5bb9
